src/components/PublicGroupList.vue

```diff
@@ -60,6 +60,7 @@
                                                     @click="$refs.monitorSettingDialog.show(group, monitor)"
                                                 />
                                             </span>
+                                            <p v-if="showCertificateExpiry" class="item-name"> Expiry: {{ formatExpiry(monitor) }} </p>
```

Comment:
Disclaimer: I am not a designer => feel free to overrule
From a design perspective, I don't like how this looks like a regular title (a user would not be able to pick up, that this is an expiry notice).

What do you think: Could we move this to a subtle or tag and still keep the current design?

Reply:
Should i try showing it as pill like the uptime percetage, with colors for showing how soon it will be expiring?
My reason for avoiding that was to not clutter the ui.

Reply:
Yes, a pill is what I meant with tag. This would be one option.

Reply:
1. Expiry inside the pill
![Screenshot 2023-07-06 at 1 36 50 PM](https://github.com/louislam/uptime-kuma/assets/12068093/36b90afa-0459-4f93-a62d-d795d2997247)

2. Expiry outside the pill
![Screenshot 2023-07-06 at 1 35 46 PM](https://github.com/louislam/uptime-kuma/assets/12068093/50888fa0-3b6e-468d-838c-40d00baca845)

Color is green for expiry of more than 7 days
Color is red for everything else

Reply:
@louislam Do you have any preference out of the above two designs?